common/evm_block_ref.go

```diff
@@ -7,6 +7,11 @@ import (
 	"strings"
 )
 
+// List of methods that static result is expected for. i.e. result will not change over time
+var EvmStaticMethods = map[string]bool{
+	"eth_chainId": true,
+}
+
 func ExtractEvmBlockReference(rpcReq *JsonRpcRequest, rpcResp *JsonRpcResponse) (string, int64, error) {
```

Comment:
is it not common to have named returned values in go? e.g it is easy to understand the requests of this function but not so much on the returned values